cmd/query/app/querysvc/adjuster/otelattribute.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package adjuster
+
+import (
+	"fmt"
+
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/ptrace"
+
+	"github.com/jaegertracing/jaeger/pkg/otelsemconv"
+)
+
+var otelLibraryKeys = map[string]struct{}{
+	string(otelsemconv.TelemetrySDKLanguageKey):   {},
+	string(otelsemconv.TelemetrySDKNameKey):       {},
+	string(otelsemconv.TelemetrySDKVersionKey):    {},
+	string(otelsemconv.TelemetryDistroNameKey):    {},
+	string(otelsemconv.TelemetryDistroVersionKey): {},
+}
+
+// OTELAttribute creates an adjuster that removes the OpenTelemetry library attributes
+// from spans and adds them to the attributes of a resource.
+func OTELAttribute() Adjuster {
+	return Func(func(traces ptrace.Traces) (ptrace.Traces, error) {
```

Comment:
@yurishkuro these adjusters are currently making modifications to the input `traces` - is that fine or do we want to make a copy of the traces before returning them?

Reply:
I think it's fine, the same happened with the v1 adjusters.

Reply:
But we could reflect that in the interface by making it not return anything.

Reply:
sounds good - i can do that in a follow-up PR